any_member.hpp

```diff
@@ -604,25 +603,20 @@ any_member<ClassType>& MemberSymbolTable<ClassType>::operator[]
     (std::string const& s
     )
 {
-    auto i = map_.find(s);
-    if(map_.end() == i)
+    auto i = std::find(member_names_.begin(), member_names_.end(), s);
+    if(member_names_.end() == i || s != *i)
```

Comment:
It doesn't seem useful to have `|| s != *i` part here, it looks like a leftover and I'm removing it in #114, please let me know if I'm missing something here.

Reply:
Agreed: it's unuseful.